Sofa/Component/Collision/Geometry/src/sofa/component/collision/geometry/LineModel.inl

```diff
@@ -565,7 +565,7 @@ void LineCollisionModel<DataTypes>::computeBBox(const core::ExecParams* params,
 {
     SOFA_UNUSED(params);
 
-    if( !onlyVisible ) return;
+    if( onlyVisible && !sofa::core::visual::VisualParams::defaultInstance()->displayFlags().getShowCollisionModels()) return;
```

Comment:
I am not sure that relying on the single instance is a good idea. Are you sure that this is the one we modify when changing the display flags? Also, it is not consistant with the `draw` method which accepts a `vparams`.